camel/agents/task_agent.py

```diff
@@ -375,3 +376,103 @@ def run(
 
         sub_tasks_msg = task_response.msgs[0]
         return get_task_list(sub_tasks_msg.content)
+
+
+class RoleAssignmentAgent(ChatAgent):
```

Comment:
Move `RoleAssignmentAgent` to a separate py file.

Reply:
Thanks, done. : )